packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx

```diff
@@ -22,10 +22,21 @@ const LanguagesForm = ({ isManageLanguageVisible = false }: Props) => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const { watch, control, register } = useFormContext<SignInExperienceForm>();
   const isAutoDetect = watch('languageInfo.autoDetect');
+  const defaultLanguage = watch('languageInfo.fallbackLanguage');
   const [isManageLanguageFormOpen, setIsManageLanguageFormOpen] = useState(false);
 
-  const { context: customPhrasesContext, Provider: CustomPhrasesContextProvider } =
-    useCustomPhrasesContext();
+  const { existedLanguageTags, setDefaultLanguageTag } = useContext(CustomPhrasesContext);
+
+  const defaultLanguageOptions = useMemo(() => {
+    return existedLanguageTags.map((languageTag) => ({
+      value: languageTag,
+      title: languages[languageTag],
+    }));
+  }, [existedLanguageTags]);
```

Comment:
nit: useMemo(() => ({}));

Reply:
nit: also feels like we could simply call it langaugeOptions

Reply:
According to @wangsijie 's advice, refactored.